ui/revenuecatui/src/test/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/VariableProcessorTest.kt

```diff
@@ -84,13 +84,22 @@ class VariableProcessorTest {
         expectVariablesResult("{{ price_per_period }}", "$67.99/yr")
         expectVariablesResult("{{ price_per_period }}", "$7.99/mth", rcPackage = TestData.Packages.monthly)
         expectVariablesResult("{{ price_per_period }}", "$1.99/wk", rcPackage = TestData.Packages.weekly)
+        expectVariablesResult("{{ price_per_period }}", "$15.99/2 mths", rcPackage = TestData.Packages.bimonthly)
+        expectVariablesResult("{{ price_per_period }}", "$23.99/3 mths", rcPackage = TestData.Packages.quarterly)
+        expectVariablesResult("{{ price_per_period }}", "$39.99/6 mths", rcPackage = TestData.Packages.semester)
+        expectVariablesResult("{{ price_per_period }}", "$1.000", rcPackage = TestData.Packages.lifetime)
     }
 
+
     @Test
     fun `process variables processes price_per_period localized in spanish`() {
         expectVariablesResult("{{ price_per_period }}", "$67.99/a", esLocale)
         expectVariablesResult("{{ price_per_period }}", "$7.99/m.", esLocale, TestData.Packages.monthly)
         expectVariablesResult("{{ price_per_period }}", "$1.99/sem.", esLocale, TestData.Packages.weekly)
+        expectVariablesResult("{{ price_per_period }}", "$15.99/2 m.", esLocale, TestData.Packages.bimonthly)
+        expectVariablesResult("{{ price_per_period }}", "$23.99/3 m.", esLocale, TestData.Packages.quarterly)
+        expectVariablesResult("{{ price_per_period }}", "$39.99/6 m.", esLocale, TestData.Packages.semester)
```

Comment:
Yeah, as you mentioned, the price here is what we get from the store and we don't format it ourselves. We could, but I think it's better to leave the original unless we have to. And this will be dots in US, so will leave as is I think

Reply:
Yes, too much to build another set of products just for the esLocale lol

Reply:
Yeah I have the same issue in some of the iOS tests, it's fine :)